### ISSUE TITLE
In package.json, cordova plugin list is never sorted

### ISSUE THREAD

**Issue by jpike88** [apache/cordova-cli#420]
# Bug Report

## Problem
In package.json, cordova plugin list is not sorted. Makes things hard to understand when adding new plugins.

![Screen Shot 2019-04-04 at 12 35 36 pm](https://user-images.githubusercontent.com/9585787/55531930-28434c80-56d6-11e9-8c6b-6d71cc96e1b4.png)

### What is expected to happen?
Object keys should be sorted alphabetically.

### What does actually happen?
They're never sorted.

### Version information

macOS High Sierra
Cordova 9.0.0

## Checklist

- [ x] I searched for existing GitHub issues
- [ x] I updated all Cordova tooling to most recent version
- [ x] I included all the necessary information above

**Comment by raphinesse** [comment]
Good point. Same goes for the list of installed platforms. Could be done using either [sort-object-keys](https://www.npmjs.com/package/sort-object-keys) or [sorted-object](https://www.npmjs.com/package/sorted-object).

@dpogue Should be added to the long list of features for https://github.com/apache/cordova-common/pull/34.

Should we move this issue to cordova-common then?

**Comment by dpogue** [comment]
Yeah... I will mention that the order of the list in package.json is the order in which `cordova prepare` restores the plugins, and I've needed to manually order that list at times to avoid weird plugin installation failures 😢 

So I'm not super keen on reordering this in package.json whenever a plugin is added, because that will be frustrating for people who have things in a specific order.

**Comment by raphinesse** [comment]
Ugh, that's horrifying :nauseated_face:

**Comment by kputh** [comment]
Not having a stable order for plugins and platforms also makes it unnessesarily difficult to understand changes in a version control system: removing an old version of a platform and adding a new one makes one line disappear and another one appear, even though only a version number changed. Whey you update many dependencies at once, the diff easily becomes incomprehensible.
